docs/substrate/constants.md

```diff
@@ -30,10 +30,10 @@ ___
 ### babe
 
 ▸ **epochDuration**: `u64`
-- **summary**:   The number of **slots** that an epoch takes. We couple sessions to  epochs, i.e. we start a new session once the new epoch begins.
+- **summary**:   The number of **slots** that an epoch takes. We couple sessions to epochs, i.e. we start a new session once the new epoch begins.
```

Comment:
This won't apply since this is generated from the metadata, it will be overridden as soon as merged. However, if you look at how we concat the docs (inside the generation script), in there we can do a `s/  / /` on all doc strings.